<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileTransfer</class>
 <widget class="QWidget" name="FileTransfer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Open_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    color: white;
    font: bold 13pt;
    border:none;
    min-height: 24px;
    min-width: 60px;
	border-image: url(:/images/images/button-login.svg);
	
}
QPushButton:hover{
    color: lightgray;
    border:none;
	border-image: url(:/images/images/button-login-hover.svg);
}
QPushButton:pressed {
    color: lightgray;
    border:none;
	border-image: url(:/images/images/button-login-press.svg);
    padding-top: -15px;
    padding-bottom: -17px;
} </string>
   </property>
   <property name="text">
    <string>打开......</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Send_pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    color: white;
    font: bold 13pt;
    border:none;
    min-height: 24px;
    min-width: 60px;
	border-image: url(:/images/images/button-login.svg);
	
}
QPushButton:hover{
    color: lightgray;
    border:none;
	border-image: url(:/images/images/button-login-hover.svg);
}
QPushButton:pressed {
    color: lightgray;
    border:none;
	border-image: url(:/images/images/button-login-press.svg);
    padding-top: -15px;
    padding-bottom: -17px;
} </string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FileName_lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
			border-image: url(:/images/images/button-quit.svg);
   			border:none;
}
QPushButton:hover{border:none;border-image: url(:/images/images/button-quit.svg);}
QPushButton:pressed{border:none;border-image: url(:/images/images/button-quit.svg);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/button-quit.svg</normaloff>:/images/images/button-quit.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>431</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="OwnAvatar_label">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="images.qrc">:/images/images/avatar.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>68</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="Status_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="transProgressBar">
        <property name="styleSheet">
         <string notr="true">QProgressBar {
    border: 5px;
	border-color: rgb(74, 181, 238);
    text-align: center;
}

QProgressBar::chunk {
    background-color: #05B8CC;
    width: 10px;
	margin: 0.5px;
}</string>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="FriendAvatar_label">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="images.qrc">:/images/images/avatar.png</pixmap>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>Open_pushButton</zorder>
  <zorder>Send_pushButton</zorder>
  <zorder>FileName_lineEdit</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>FileTransfer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
